mp/src/game/server/momentum/mom_triggers.cpp

```diff
@@ -22,6 +22,28 @@ static MAKE_TOGGLE_CONVAR(mom_triggers_overlay_bbox_enable, "0", FCVAR_DEVELOPME
 static MAKE_TOGGLE_CONVAR(mom_triggers_overlay_text_enable, "0", FCVAR_DEVELOPMENTONLY,
                           "Toggles showing the entity text for momentum triggers, needs map restart if changed!\n");
 
+CON_COMMAND(mom_trigger_teleport_progress, "Teleports player to the last progress trigger\n")
```

Comment:
To keep the naming of progress related commands I suggest renaming this to something like `mom_prog_tele`, and moving it into [`mom_system_progress.cpp`](https://github.com/momentum-mod/game/blob/e3efc48eb58cf56af61b2fef8d0bf7e3bf754680/mp/src/game/server/momentum/mom_system_progress.cpp).

Reply:
A good suggestion I have renamed it and moved to mom_system_progress.cpp